service_extensions/callouts/add_header/noxfile_config.py

```diff
@@ -22,7 +22,7 @@
 
 TEST_CONFIG_OVERRIDE = {
     # Supported versions are >= 3.7 as per https://pypi.org/project/grpcio/
-    "ignored_versions": ["2.7", "3.6"],
+    "ignored_versions": ["2.7", "3.6", "3.7", "3.8", "3.10", "3.11"],  # "3.9"
```

Comment:
suggestion: we should be testing oldest (3.8) and newest (3.11) supported version.

Reply:
It was for quick temporary check to see if the new ports are accepted (using 8443 instead of port 443). The latest config includes looks like `"ignored_versions": ["2.7", "3.6", "3.7"]`